index.js

```diff
@@ -50,8 +49,11 @@ function test(p, opts) {
   });
 }
 
+exports.test = test;
```

Comment:
__*Question*__

Why do you need to expose the test function if you've added `contextName` to the options below?  I'd prefer to keep it private to limit the exposed API.